aten/src/ATen/native/TensorShape.cpp

```diff
@@ -762,7 +762,12 @@ Tensor repeat(const Tensor& self, IntArrayRef repeats) {
 
   Tensor xtensor = self.expand(padded_size);
 
-  Tensor result = at::empty(target_size, self.options());
+  Tensor result;
+  if (self.is_quantized()) {
+    result = at::empty_quantized(target_size, self);
+  } else {
+    result = at::empty(target_size, self.options());
```

Comment:
does dispatching on tensor type not work for `at::empty`?

Reply:
yeah, quantized parameters are not included in TensorOptions(self.options())